### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by notwaldorf**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -23,18 +23,36 @@
...
-    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
-    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
-    "iron-icons": "PolymerElements/iron-icons#^1.0.0",
-    "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+    "iron-component-page": "PolymerElements/iron-component-page#2.0-preview",
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#2.0-preview",
+    "iron-icons": "PolymerElements/iron-icons#2.0-preview",
+    "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
```
Comment (bicknellr) [inline bower.json]: WCT 4 & 6 have the right versions of test-fixture now.

**Comment by notwaldorf** [comment]
@bicknellr this is ready for 👀

File: bower.json (hunk 1 above)
Comment (notwaldorf) [inline bower.json]: done

**Comment by notwaldorf** [comment]
@bicknellr done

**Pull request merged**